api-docs/api-reference/methods/tenants/get-list-tenants-v2.0.rst

```diff
@@ -7,12 +7,18 @@ List tenants
 
     GET /v2.0/tenants
 
-Get a list of tenants, or look up a tenant by name.
+Get a list of tenants.
 
-Identity administrators can use this operation to retrieve a list of tenants
-or an individual tenant by name.
+The list of tenants returned is determined by the X-Auth-Token provided on the
+request. The list will include tenants a user has access to. The tenants the
+user has access to via RCN roles will only be returned if requested.
 
```

Comment:
Replace lines 12-14 with the following:
The list of tenants returned includes only those to which the user has access, based on the user's X-Auth-token. The tenants to which the user has access via RCN roles are returned only if requested.